{
    "config": {
        "abort": {
            "already_configured": "Konto har redan konfigurerats",
            "reauth_failed_existing": "Det gick inte att uppdatera konfigurationsposten, ta bort integrationen och konfigurera den igen.",
            "reauth_successful": "\u00c5terautentisering lyckades",
            "unknown": "Ov\u00e4ntat fel"
        },
        "error": {
            "cannot_connect": "Det gick inte att ansluta.",
            "invalid_api_key": "Ogiltig API-nyckel",
            "not_main_key": "Fel API-nyckeltyp uppt\u00e4ckt, anv\u00e4nd \"huvud\" API-nyckeln",
            "reauth_failed_matching_account": "API-nyckeln du angav matchar inte konto-id:t f\u00f6r befintlig konfiguration.",
            "unknown": "Ov\u00e4ntat fel"
        },
        "step": {
            "reauth_confirm": {
                "data": {
                    "api_key": "API-nyckel"
                },
                "description": "Du m\u00e5ste ange en ny \"huvud\" API-nyckel fr\u00e5n UptimeRobot",
                "title": "Autentiseringen har upph\u00f6rt att g\u00e4lla f\u00f6r {name}"
            },
            "user": {
                "data": {
                    "api_key": "API-nyckel"
                },
                "description": "Du m\u00e5ste ange \"huvud\" API-nyckeln fr\u00e5n UptimeRobot"
            }
        }
    },
    "entity": {
        "sensor": {
            "monitor_status": {
                "state": {
                    "down": "Ner",
                    "not_checked_yet": "Inte kontrollerad \u00e4nnu",
                    "pause": "Pausa",
                    "seems_down": "Verkar nere",
                    "up": "Upp"
                }
            }
        }
    }
}